test/spawn.jl

```diff
@@ -248,6 +248,16 @@ let bad = "bad\0name"
     @test_throws ArgumentError run(setenv(`echo hello`, "good"=>bad))
 end
 
+# issue #12829
+let out = Pipe()
+    @test_throws ArgumentError write(out, "not open error")
+    open(`cat`, "w", out) do io
```

Comment:
note mostly to myself that we won't always have `cat` available in the windows binaries post-libgit2, but we can use busybox or something to keep this and similar tests passing